config/env/cloud-foundry.js

```diff
@@ -3,7 +3,9 @@
 var cfenv = require('cfenv'),
   appEnv = cfenv.getAppEnv(),
   cfMongoUrl = appEnv.getService('mean-mongo') ?
-  appEnv.getService('mean-mongo').credentials.uri : undefined;
+  appEnv.getService('mean-mongo').credentials.uri ? 
```

Comment:
@ilanbiala would you prefer I broke it up into a traditional if else so it is clearer?